### PR TITLE
 Vet-Clinic-Database

### PR DESCRIPTION
**Pull request by AhmadiGu**
### Database base diagram

*The following changes introduced:*

- Take a look at the following database diagram for a clinic
- Create a file named schema_based_on_diagram.sql 
- Join tables from many-to-many relationships might not appear in the diagram
- Add the FK indexes

### CODE REVIEW

File: schema_based_on_diagram.sql [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+
+CREATE TABLE treatment (
+    id INT NOT NULL GENERATED ALWAYS AS IDENTITY PRIMARY KEY,
+    name varchar(200),
+    type varchar(200)
+);
+
+CREATE TABLE medical_histories (
+    id INT NOT NULL GENERATED ALWAYS AS IDENTITY PRIMARY KEY,
+    attmat_at timestamp,
```
Comment (zieeco) [inline schema_based_on_diagram.sql]: - Please, let's kindly correct our spelling as this should be `admitted_at`.

File: LICENSE [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+The above copyright notice and this permission notice shall be included in all
+copies or substantial portions of the Software.
+
+THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+SOFTWARE.
```
Comment (zieeco) [inline LICENSE:21]: - Hey, great adding the readme file, however, your readme is not close to being professional as you've not added all the important header sections as required. Please, kindly update your readme file using this [template](https://github.com/microverseinc/readme-template) and add the remaining sections to make your readme more professional.

**Review by zieeco — CHANGES_REQUESTED** [review]
Hi @AhmadiGu ,


Great job so far! 👏 
You've done well, however, there are some issues that you still need to work on to go to the next project, but you are almost there!


## To highlight 🎯

- No linter errors.✔️
- Nice shcehma file. ✔️
- Descriptive PR. ✔️


## Required Changes ♻️
_Check the comments under the review._





## Optional suggestions
- N/A


**_Every comment with the **[OPTIONAL]** prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._**


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by prolajumokeoni — APPROVED** [review]
Hi @AhmadiGu 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 
## Highlights
- No linter errors ✅ 
- Git flow was used correctly 🚀 
- Good design 💯 
- Good Pr title and description 💪🏿 

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


-  Kindly add the key features and future features section of your app to your readme, all sections from the microverse template should be present in your readme and the content personalized 👍🏿 


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification @prolajumokeoni.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**